### PR TITLE
Added Render-Atlanta 2022

### PR DESCRIPTION
**Pull request by ThugDebugger**

### CODE REVIEW

**Comment by ThugDebugger** [comment]
BEGIN THE HYPE TRAIN!!!!

File: site/conferences/2022-render-atl [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Render-Atlanta (RenderATL) 2022
+url: https://www.renderatl.com/
+cocUrl: https://www.renderatl.com/code-of-conduct
```
Comment (chriscoyier) [inline site/conferences/2022-render-atl]: This is a 404 page. Have y'all established one yet?

**Comment by ThugDebugger** [comment]
Hey Chris,

It should redirect shortly. It’s the same one from 2021 if you view under our past conference. 

	
Justin E. Samuels
a: P.O. Box 15256  | Atlanta, GA 30333
e: ***@***.*** | w: www.ThugDebugger.dev
p: +1-662-617-0825

File: site/conferences/2022-render-atl (hunk 1 above)
Comment (ThugDebugger) [reply site/conferences/2022-render-atl]: Hey Chris! 

It's been updated and [can be viewed here](https://www.renderatl.com/thecode#/)

**Comment by ThugDebugger** [comment]
Code of Conduct updated and [available here](https://www.renderatl.com/thecode#/)

**Comment by chriscoyier** [comment]
sweet thanks!

**Pull request merged**